snap/snapcraft.yaml

```diff
@@ -39,6 +44,14 @@ apps:
       XDG_CONFIG_HOME: &daemon-config $SNAP_DATA/config
       DAEMON_CONFIG_HOME: *daemon-config # temporary
     daemon: simple
+    plugs:
```

Comment:
We're gonna need to include the `libvirt` plug as well, right?

Reply:
Quite right, added